lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/file/download/v2/DownloadListDAOImpl.java

```diff
@@ -707,16 +720,19 @@ public Long addChildrenToDownloadList(Long userId, Long parentId, boolean useVer
 	public JSONObject getItemManifestDetails(DownloadListItem item) {
 		String sql = null;
 		if (item.getVersionNumber() == null) {
-			sql = "SELECT " + ManifestKeys.buildSelect() + ", R." + COL_REVISION_USER_ANNOS_JSON + " FROM " + TABLE_NODE
+			sql = "SELECT " + ManifestKeys.buildSelect() + ", R." + COL_REVISION_USER_ANNOS_JSON + ", DA." + COL_DERIVED_ANNOTATIONS_ANNOS + " FROM " + TABLE_NODE
 					+ " N JOIN " + TABLE_REVISION + " R ON (N." + COL_NODE_ID + " = R." + COL_REVISION_OWNER_NODE
 					+ " AND N." + COL_NODE_CURRENT_REV + " = R." + COL_REVISION_NUMBER + ") JOIN " + TABLE_FILES
-					+ " F ON (R." + COL_REVISION_FILE_HANDLE_ID + " = F." + COL_FILES_ID + ") WHERE N." + COL_NODE_ID
+					+ " F ON (R." + COL_REVISION_FILE_HANDLE_ID + " = F." + COL_FILES_ID + ") "
+					+ " LEFT JOIN " + TABLE_DERIVED_ANNOTATIONS + " DA ON (N." + COL_NODE_ID + " = DA." + COL_DERIVED_ANNOTATIONS_ID + ")"
+					+ " WHERE N." + COL_NODE_ID
 					+ " = :synId";
 		}else {
-			sql = "SELECT " + ManifestKeys.buildSelect() + ", R." + COL_REVISION_USER_ANNOS_JSON + " FROM " + TABLE_NODE
```

Comment:
we are still trying to decide what to do with versions.  For now we are not going to include derived when a specific version is given.